hi-ml-cpath/src/health_cpath/utils/plots_utils.py

```diff
@@ -311,11 +372,18 @@ def save_plots(self, outputs_dir: Path, tiles_selector: Optional[TilesSelector],
             )
             figures_dir = make_figure_dirs(subfolder="fig", parent_dir=outputs_dir)
 
+            if self.stratify_plots_by is not None:
```

Comment:
you're doing that check already in the get_metadata functin?

Reply:
good catch, thanks, removed from here